lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -4868,6 +4974,10 @@ ParseResult FIRCircuitParser::parseToplevelDefinition(CircuitOp circuit,
     return emitError(getToken().getLoc(), "only modules may be public");
   case FIRToken::kw_type:
     return parseTypeDecl();
+  case FIRToken::kw_option:
+    if (requireFeature(nextFIRVersion, "options"))
```

Comment:
I'd go with the full feature name here or possibly both: 
```suggestion
    if (requireFeature(nextFIRVersion, "option groups/instance choices"))
```

Reply:
Also, new features usually lock in this check in a test.